operations/secure_random/test_secure_random.py

```diff
@@ -9,15 +9,18 @@
 dirname = os.path.dirname(tfe.__file__)
 shared_object = dirname + '/operations/secure_random/secure_random_module.so'
 secure_random_module = tf.load_op_library(shared_object)
+seeded_random_uniform = secure_random_module.seeded_random_uniform
+random_uniform = secure_random_module.secure_random_uniform
```

Comment:
rename `secure_random_module.secure_random_uniform` to `secure_random_module.random_uniform` to match the seeded version?

Reply:
Tensorflow wouldn't let me do that. I don't think you can have ops with the same name even though they're in a different shared object.

Reply:
ah I see, makes sense. is it worth using `secure` everywhere then for consistency?

Reply:
sure, we're exposing the shorter name to the user, `seeded_random_uniform` and `random_uniform`